### PR TITLE
Made HW21, divided ui and api tests.

### PR DESCRIPTION
**Pull request by anastasiya-harbatsevich**
Made HW21

Divide ui and api tests to different modules

Write test using Selenide

### CODE REVIEW

File: ui-tests/pom.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```
Comment (alexshamrai) [inline ui-tests/pom.xml]: Do you use org.apache.commons here? Why do you need it?

File: ui-tests/src/main/java/com/practicesoftwaretesting/utils/SelectorUtils.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    public static By byDataTest(String attributeValue) {
+        return by("data-test", attributeValue);
+    }
+
+    public static By byFor(String attributeValue) {
+        return by("for", attributeValue);
+    }
+
+    public static By byId(String attributeValue) {
```
Comment (alexshamrai) [inline ui-tests/src/main/java/com/practicesoftwaretesting/utils/SelectorUtils.java]: Do not reinvent the bicycle
Read the documentation, please. If you think of any improvement, please check if somebody has already done it.
https://selenide.org/javadoc/current/com/codeborne/selenide/Selectors.html#byId(java.lang.String)

File: ui-tests/src/main/java/com/practicesoftwaretesting/asserts/RegisterPageAsserts.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package com.practicesoftwaretesting.asserts;
+
+import static com.codeborne.selenide.CollectionCondition.size;
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$;
+import static com.practicesoftwaretesting.utils.SelectorUtils.byFor;
+import static com.practicesoftwaretesting.utils.SelectorUtils.byId;
```
Comment (alexshamrai) [inline ui-tests/src/main/java/com/practicesoftwaretesting/asserts/RegisterPageAsserts.java]:
```suggestion
import static com.codeborne.selenide.Selectors.byId;
```

**Review by alexshamrai — CHANGES_REQUESTED** [review]

File: ui-tests/pom.xml (hunk 1 above)
Comment (anastasiya-harbatsevich) [reply ui-tests/pom.xml]: Yes, it is not necessary. Removed it.

File: ui-tests/src/main/java/com/practicesoftwaretesting/utils/SelectorUtils.java (hunk 2 above)
Comment (anastasiya-harbatsevich) [reply ui-tests/src/main/java/com/practicesoftwaretesting/utils/SelectorUtils.java]: It was made by mistake, thanks for comment

File: ui-tests/src/main/java/com/practicesoftwaretesting/asserts/RegisterPageAsserts.java (hunk 3 above)
Comment (anastasiya-harbatsevich) [reply ui-tests/src/main/java/com/practicesoftwaretesting/asserts/RegisterPageAsserts.java]: replaced

**Review by alexshamrai — APPROVED** [review]

**Pull request merged**